### PR TITLE
core: webrtc calls api types

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: src/Simplex/Chat/Controller.hs [hunk 1]
```diff
@@ -240,6 +248,12 @@ data ChatResponse
   | CRPendingSubSummary {pendingSubStatus :: [PendingSubStatus]}
   | CRSndFileSubError {sndFileTransfer :: SndFileTransfer, chatError :: ChatError}
   | CRRcvFileSubError {rcvFileTransfer :: RcvFileTransfer, chatError :: ChatError}
+  | CRCallInvitationSent {contact :: Contact}
```
Comment (jr-simplex) [inline src/Simplex/Chat/Controller.hs:251]: Is there a reason to use the specific response for invitation and not CRCmdOk like for other commands?

File: src/Simplex/Chat/Messages.hs [hunk 2]
```diff
@@ -524,13 +537,42 @@ dbJsonCIContent = \case
...
   DBJCISndMsgContent mc -> ACIContent SMDSnd $ CISndMsgContent mc
   DBJCIRcvMsgContent mc -> ACIContent SMDRcv $ CIRcvMsgContent mc
   DBJCISndDeleted cidm -> ACIContent SMDSnd $ CISndDeleted cidm
   DBJCIRcvDeleted cidm -> ACIContent SMDRcv $ CIRcvDeleted cidm
+  DBJCISndCall {status, duration} -> ACIContent SMDSnd $ CISndCall status duration
+  DBJCIRcvCall {status, duration} -> ACIContent SMDRcv $ CIRcvCall status duration
+
+data CICallStatus
+  = CISCallPending
+  | CISCallMissed
```
Comment (jr-simplex) [inline src/Simplex/Chat/Messages.hs:554]: How is Missed status achieved in state machine? Is it when the call wasn't successfully negotiated? Would it be the same when the call couldn't be negotiated due to an error?

File: src/Simplex/Chat/Controller.hs (hunk 1 above)
Comment (epoberezkin) [reply src/Simplex/Chat/Controller.hs:251]: callId was used internally in the beginning, can be replaced with Ok too

File: src/Simplex/Chat/Messages.hs (hunk 2 above)
Comment (epoberezkin) [reply src/Simplex/Chat/Messages.hs:554]: When there was no response and it ended. Probably there should be one more status for the error

**Review by jr-simplex — APPROVED** [review]

**Pull request merged**